tools/hardhat-example/hardhat.config.js

```diff
@@ -46,12 +28,31 @@ task('contract-call', async (taskArgs) => {
   return contractCall(taskArgs.contractAddress, taskArgs.msg);
 });
 
+/** @type import('hardhat/config').HardhatUserConfig */
 module.exports = {
-  solidity: '0.8.4',
-  defaultNetwork: 'relay',
+  mocha: {
+    timeout: 3600000,
+  },
+  solidity: {
+    version: '0.8.9',
+    settings: {
+      optimizer: {
+        enabled: true,
+        runs: 500,
+      },
+    },
+  },
+  defaultNetwork: 'local',
   networks: {
-    relay: {
-      url: process.env.RELAY_ENDPOINT
-    }
-  }
+    local: {
+      url: process.env.RELAY_ENDPOINT,
+      accounts: [process.env.OPERATOR_PRIVATE_KEY, process.env.RECEIVER_PRIVATE_KEY],
+      chainId: 298,
+    },
+    testnet: {
+      url: 'https://testnet.hashio.io/api',
+      accounts: [process.env.OPERATOR_PRIVATE_KEY_TESTNET, process.env.RECEIVER_PRIVATE_KEY_TESTNET],
```

Comment:
Leave this array empty.
Let's not add another config just for testnet. Let devs customize this part or reuse the same config name with the intent they'd fill it in appropriately as needed

Reply:
You're right. Left it empty array.